src/components/Launch/LaunchWorkflowForm/SearchableSelector.tsx

```diff
@@ -127,18 +129,21 @@ function useSearchableSelectorState<DataType>({
     };
 
     const onFocus = () => {
+        setIsExpanded(false);
+        setHasReceivedInput(false);
```

Comment:
Does order here matter? Or do the hooks complete asynchronously?

Reply:
My understanding is that state updates triggered in response to an event are batched. So the order here won't matter. But, even if that turns out not to be the case, the order here is alright.